.ci/azure/pipelines/obs-packaging-ci.yml

```diff
@@ -11,9 +11,37 @@ pool:
   vmImage: 'ubuntu-latest'
 
 steps:
-- script: echo setup
-  displayName: 'setup CI'
+# azure docker installation task
+# https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/tool/docker-installer?view=azure-devops 
+- task: DockerInstaller@0
+  displayName: 'Install Docker'
 
-- script: |
-    echo This is an initial CI job
-  displayName: 'Run CI'
+- bash: |
+   sudo apt-get update
+   sudo apt-get install -y git
+   git config --global user.email "azure-pipeline@kata.io"
+   git config --global user.name "azure-pipeline"
+  displayName: 'Setup'
+
+- bash: |
+   # This is set by azure, but we use the same variable
+   export AGENT_VERSION=""
+   script -qefc bash -c '.ci/packaging/request_to_obs.sh'
```

Comment:
There are a few calls to `script(1)` on this PR. As such, it might be worth specifying a transcript file name for each call. That way, if "step n+1" fails (which will be logged), but the failure was actually caused by something odd that happened in "step n" (which succeeded, but whose transcript file has now been overwritten), you'll be able to debug since you'll have all the transcript files.

Reply:
I dont really use script to get the output but just to ensure there is a tty for docker run etc... So I think I will skip this suggestion this time.